tests/integration/utils.py

```diff
@@ -210,22 +210,25 @@ def __check_attributes_schema(actual_attributes, expected_attributes):
 
 # Check response of an endpoint against path cases for schema, data validity
 def test_path_request(self, endpoint, resource, response_code, test_cases,
-                      test_assertion=None):
+                      param=None, test_assertion=None):
+    schema = get_resource_schema(self, resource)
     for test_case in test_cases:
-        test_request(self, endpoint + "/" + test_case, resource, response_code,
-                     "", "", test_assertion)
+        response = make_request(self, endpoint + "/" + test_case,
```

Comment:
instead of concatenating string by using `+`, can you use an f-string to format the string?